paper/paper.md

```diff
@@ -86,10 +88,11 @@ bibliography: paper.bib
 
 
 # Summary
-The General Molecular Simulation Object, or GMSO, stands as an open-source Python data structure, offering a versatile and expandable framework for handling chemical and biomolecular topologies. This library is an integral component of the Molecular Simulation Design Framework (MoSDeF), dedicated to streamlining the creation, parameterization, and representation of systems for molecule simulations. The GMSO library serves as a dynamic repository for storing chemical/biomolecular structures, encompassing metadata, coordinates, and interaction potentials. Moreover, the library includes routines for editing and exporting stored structures into various file formats, which can be used with other software for visualization (e.g., VMD[@humphrey1996vmd] and OVITO[@]) or conducting molecular simulations (e.g., GROMACS [@abraham2015gromacs], LAMMPS[@thompson2022lammps], GOMC[@nejahi2021update]).
+The General Molecular Simulation Object, or GMSO, offers an open-source Python data structure, offering a versatile and expandable framework for handling chemical and biomolecular topologies. This library is an core component of the Molecular Simulation Design Framework (MoSDeF), dedicated to streamlining the creation, parameterization, and representation of systems for molecular simulations. The GMSO library serves as a dynamic repository for storing chemical/biomolecular structures, encompassing metadata, coordinates, and interaction potentials. Moreover, the library includes routines for editing and exporting stored structures into various file formats, which can be used with other software for visualization (e.g., VMD[@humphrey1996vmd] and OVITO[@]) or conducting molecular simulations (e.g., GROMACS [@abraham2015gromacs], LAMMPS[@thompson2022lammps], GOMC[@nejahi2021update], and HOOMD-blue[@anderson2020hoomd]).
```

Comment:
If we're going with `core` here, we should change `an` to `a`

Reply:
> The General Molecular Simulation Object, or GMSO, offers an open-source Python data structure, offering a versatile and expandable framework for handling chemical and biomolecular topologies.

I think just saying "The General Molecular Simulation Object, or GMSO, **_is_** an open-source Python data structure" sounds more concise and accurate than "stands as an open source Python data structure" or "offers an open source Python data structure" especially since the second part of the sentence talks about what it offers.

Reply:
I agree that maybe offers isn't quite the right word. My only nit-picky issue is that GMSO is not a data structure, GMSO is a package. Topology is the data structure, if we're being technical on the software language. 

Maybe something like:
GMSO
(contains, provides, presents, incorporates, consists of)
an open-source Python data structure

However, I think the key for this sentence is what GMSO **is**, as Chris emphasizes. So maybe a better change would be: "or GMSO, is an open-source Python package for supplementing molecular simulation workflows."

Reply:
I like the last suggestion: "GMSO, is an open-source Python package for supplementing molecular simulation workflows.", or something a long that line